MayaPkg/PluginInstallerDll/PluginInstallerDll.cpp

```diff
@@ -129,11 +128,7 @@ void installBoto3()
 
 		SetCurrentDirectoryA(fullPath.c_str());
 
-		std::string cmdScript =
-			"curl \"https://bootstrap.pypa.io/2.7/get-pip.py\" -o \"get-pip.py\" && "
-			"mayapy get-pip.py &&"
-			"mayapy -m pip install boto3\n"
-			;
+		std::string cmdScript = "mayapy -m pip install boto3\n";
```

Comment:
We don't need to install pip on Maya's python anymore?
(also we seem to be doing this for Mac)

Reply:
We need to install but only for maya 2022

Reply:
yeah no pip should be installed, and boto3 only for maya 2022

Reply:
Done